README.md

```diff
@@ -43,6 +45,16 @@ const html = markdown.require('./foo.md')
 const html = '<h1>foo</h1>'
 ```
 
+Using ESM:
+
+```js
+import html from './foo.md'
+
+// yield:
+
+const html = '<h1>foo</h1>'
+```
+
```

Comment:
I would like it to work in [this way](https://github.com/kentcdodds/babel-plugin-preval#import-comment) since some may not want every `import` statement to be transformed

Reply:
You mean notating the `import` with a comment inferring to parse that specific one differently? That could work, but currently we're looking at the extension so it might make the traversal marginally faster?

Reply:
yeah but sometimes we don't want all `.md` to be transformed by this plugin, maybe it could be an option: `alwaysTransformImports: true`, and by default we only transform import statements with a comment notation `/* markdown */`.